### PR TITLE
feat: add Show Page Emails tab

### LINKED ISSUE #2926: Create a new message tab
Create a Message tab next to Timeline, Tasks and Notes

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=15563%3A46189&mode=dev

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #2926, Closes #2927

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx [hunk 1]
```diff
@@ -104,6 +105,7 @@ export const ShowPageRightContainer = ({
       {activeTabId === 'tasks' && <EntityTasks entity={entity} />}
       {activeTabId === 'notes' && <Notes entity={entity} />}
       {activeTabId === 'files' && <Attachments targetableEntity={entity} />}
+      {activeTabId === 'emails' && <Emails />}
```
Comment (Weiko) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:108]: Can we hide this behind the messaging featureFlag?

Comment (thaisguigon) [reply packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:111]: Done! The "Emails" tab will be disabled if `isMessagingEnabled` is false.

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx [hunk 2]
```diff
@@ -52,6 +54,8 @@ export const ShowPageRightContainer = ({
   notes,
   emails,
 }: ShowPageRightContainerProps) => {
+  const isMessagingEnabled = useIsFeatureEnabled('IS_MESSAGING_ENABLED');
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:57]: Can we implement a string literal or enum for those feature flags ?

Comment (thaisguigon) [reply packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:57]: Done! I added the `FeatureFlagKey` type.

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**